styleguide/components/11-base/fonts/fonts.twig

```diff
@@ -1,7 +1,6 @@
 <h3 id="styleguide-fonts" class="styleguide-title">{{ 'Fonts' }}</h3>
 <div class="preview-inner-container styleguide-secondary-colors">
-  <p>{{ 'Base font-size (mobile, until 1920px): 16px'}}</p>
-  <p>{{ 'Base font-size (large desktop, starting from 1920px): 20px'}}</p>
+  <p>{{ 'Base font-size is 20px'}}</p>
```

Comment:
Isn't the base font-size 16px? 🤔(Detail layout is 18px)

Reply:
We should clarify that the technical base font-size is 20px and that the base font-size of the body is .8rem (or .9... I honestly don't know)